tests/calculators/test_torsion_calculators.py

```diff
@@ -0,0 +1,35 @@
+from pytest import approx
+import stko
+import stk
+from examples.torsion_example import get_torsion_info_angles
+
+
+def test_constructed_molecule():
+    """confirm torsions are appropriately mapped from constructed molecules to building blocks"""
+    
+    def test_polymer(bb1: stk.BuildingBlock, bb2: stk.BuildingBlock):
+        """attach bb1 to bb2 and test torsion mapping"""
+        polymer = stk.ConstructedMolecule(
+            stk.polymer.Linear(
+                building_blocks=(bb1, bb2),
+                repeating_unit="AB",
+                orientations=[0, 0],
+                num_repeating_units=1
+            )
+        )
+
+        tors_calculator = stko.ConstructedMoleculeTorsionCalculator()
+        tors_results = tors_calculator.get_results(polymer)
+        
+        angles = get_torsion_info_angles(polymer, list(tors_results.get_torsion_infos())[0])
+        assert angles[0] == approx(angles[1])
```

Comment:
I do not immediately see why this test is meaningful. I.e. why are the angles defined to be the same for the polymer? I thought you would want to test that the angles in the polymer are equivalent to their angles within the initial building blocks?